### ISSUE TITLE
php console serve command

### ISSUE THREAD

**Issue by Jaspervv** [themosis/documentation#69]
It might be worth documenting the `php console serve` command, it's a nice alternative if you don't use Homestead.  
[Here](https://laravel.com/docs/5.7/installation#installing-laravel) is how Laravel documents it on their Installation page.

**Comment by jlambe** [comment]
Indeed but in my tests, you still need to create a MySQL database on the side to make it work properly.

**Comment by Jaspervv** [comment]
Yes, unlike using something like Homestead it doesn't provide a full development environment.  
But it's a lot easier than having to manage a bunch of vhost configuration files, and a lot cleaner than putting all your projects in `xampp/htdocs` and getting ugly urls like `localhost/htdocs/themosis`.
